lib/recurly/token.js

```diff
@@ -63,6 +65,10 @@ export const FIELDS = [
  * @param {String} [city]
  * @param {String} [state]
  * @param {String|Number} [postal_code]
+ * @param {String} [phone]
+ * @param {String} [vat_number]
+ * @param {String} [tax_identifier]
+ * @param {String='cpf'} [tax_identifier_type]
```

Comment:
Let's remove the default value here since on the routine level its default is `undefined`